.travis.yml

```diff
@@ -24,9 +24,4 @@ before_script:
   - sh -e /etc/init.d/xvfb start
   - sleep 3
 script:
-  - npm run lint
-  - npm test
-  - npm run build:prod
-  - npm run server:prod:ci &
-  - sleep 3
-  - npm run e2e
+  - npm run ci
```

Comment:
Also, after I got AoT working I hit a hitch where Angular was crashing on startup with the production build, even though the build was passing. That's why it is important to test both for this repo's CI. Individual developers can use `npm run ci:aot` for lint, karma, and test of AoT build.